Source/WTF/wtf/JSONValues.cpp

```diff
@@ -204,10 +204,20 @@ bool parseStringToken(const CodeUnit* start, const CodeUnit* end, const CodeUnit
     return false;
 }
 
+template<typename CodeUnit>
+static inline bool isJSONWhitespace(CodeUnit character)
+{
+    // This is different from isASCIIWhitespace: JSON does not accept \v as a space.
```

Comment:
Same as `isHTTPSpace`. Should we come up with a name for this variation so we can use a single version in `ASCIICType.h` instead of two? I’m sure it will recur somewhere else too eventually.

Reply:
I am surprised that JavaScript is so different from JSON, in other words how `Lexer::isWhiteSpace` is so different.

Reply:
That's good! But not sure about the best name, so for now, I'll name it as isJSONOrHTTPWhitespace.